README.md

```diff
@@ -1,2 +1,64 @@
+| nes_vaccination_record/vaccination_management/medication/batch_id | Vaccine batch code |
+| nes_vaccination_record/vaccination_management/medication/expiry | Vaccine expiration data. *Note*: we don't record this (VMT might have it) |
+| nes_vaccination_record/vaccination_management/medication_supply_amount/amount_description | Amount given. *Note*: NCDS don't record this yet, probably will in the future |
+| nes_vaccination_record/vaccination_management/reason:0 | Map to the FHIR Immunization.statusReason element when ISM is ‘Vaccine dose not given’ and to ‘Immunization.reasonCode’ when ISM is ‘Vaccine dose administered’. *Suggestion*: we might want to record the SNOMED code for reason also? |
+| nes_vaccination_record/vaccination_management/administration_details/route | FHIR code - https://www.hl7.org/fhir/valueset-immunization-route.html (NCDS have a longer list) |
+| nes_vaccination_record/vaccination_management/administration_details/body_site | FHIR code - https://www.hl7.org/fhir/valueset-immunization-site.html (NCDS have a longer list) |
+| nes_vaccination_record/vaccination_management/nes_vaccination_event_details/vaccination_administration_purpose|code | Either booster dose (at0009) or primary (at0008) |
+| nes_vaccination_record/vaccination_management/dose_number" | Dose number |
+| nes_vaccination_record/vaccination_management/note | *Note*: supported but may not be used in NCDS |
+| nes_vaccination_record/vaccination_management/appointment_id | 14ebf56b-8d76-439b-8bfe-884e1e8fa569 |
+| nes_vaccination_record/vaccination_management/appointment_id\|issuer | TODO REMOVE? Issuer |
+| nes_vaccination_record/vaccination_management/appointment_id\|assigner | TODO REMOVE? Assigner |
+| nes_vaccination_record/vaccination_management/appointment_id\|type | TODO REMOVE? Prescription |
+| nes_vaccination_record/vaccination_management/nes_vaccination_protocol/target_disease:0 | Disease SNOMED code |
+| nes_vaccination_record/vaccination_management/nes_vaccination_protocol/country_of_vaccination\|code | Country code |
+| nes_vaccination_record/vaccination_management/nes_vaccination_protocol/country_of_vaccination\|value | Country value |
+| nes_vaccination_record/vaccination_management/nes_vaccination_protocol/country_of_vaccination\|terminology | Country ISO |
+| nes_vaccination_record/vaccination_management/nes_vaccination_protocol/vaccination_centre | Vaccination centre name *Question*: overlap with health care facility at top? |
+| nes_vaccination_record/vaccination_management/time | Time composition created |
+
+## How to use the composition
+
+We can create a new composition in EHRbase using the below cURL command or our [postman collection](https://nesdigital.atlassian.net/wiki/spaces/NDSENG/pages/130383926). Note that
+ - you will have to create an EHR before you can create a composition (see the [postman collection](https://nesdigital.atlassian.net/wiki/spaces/NDSENG/pages/130383926) if your unsure how to do this)
```

Comment:
@Kris-Mcalpine I updated the readme to mention that it's necessary to create an EHR and pointed them towards the postman collection.

I edited the original commit and force pushed rather than me forget to squash them on merge 😄